File: Others/sort_asc.js

const contoh = [10, 2, 5, 7, 8, 10, 11, 20, 99, 1, 0, 5, 4];

for (let i = 0; i < contoh.length; i++) {
  min_index = i;
  // check all elements, whether they're small than element with current min index or not
  for (let j = i + 1; j < contoh.length; j++) {
    if (contoh[j] < contoh[min_index]) {
      // search element that less than element with min_index
      min_index = j;
    }
  }
  
  // swap the element's position
  let temp =contoh[min_index];
  contoh[min_index] = contoh[i];
  contoh[i] = temp;
}

console.log(contoh);

File: Others/sort_asc.py
# sorting array (ascending) using selection sort algoritm

def sort_asc(array: list):
	for i in range(len(ex)):

		# current min index
		min_index = i
		
		for j in range(i+1, len(ex)):
			# check all elements, whether their small than current min index 
			if array[j] < array[min_index]:
				min_index = j

		# swap the element with current index (i) with the element with min index 
		array[i], array[min_index] = array[min_index], array[i]
	
	return array

